extras/openvino-operator-openshift/openshift-bundle/0.2.0/metadata/annotations.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
-  operators.operatorframework.io.bundle.package.v1: openvino-operator
+  operators.operatorframework.io.bundle.package.v1: ovms-operator
```

Comment:
Is this change intentional? Shouldn't it be `openvino-operator`?

Reply:
@mzegla it is intentional, otherwise we would need to create new offering in RH catalogue. It was agreed to do it for golang version later